### PR TITLE
clean up

### PR DESCRIPTION
**Pull request by aftabsiddiqui**
renamed folder and cleaned up readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,44 @@ This is a Python 3 port of the utility functions from Dragon Research Labs
...
 than anything else.
+
+Installation and Usage instructions.
+
+```shell
+$ sudo git clone https://github.com/aftabsiddiqui/py3-rpki-utils.git
+$ cd py3-rpki-utils
+$ sudo python3 setup.py build
+$ cp build/*/*/*/_POW*.so rpki/POW/_POW.so
+$ cd print
```
Comment (ppaeps) [inline README.md:15]: Any particular reasons for the `sudo`?  This should not require privilege.

**Review by ppaeps — CHANGES_REQUESTED** [review]
This looks good in principle.  I would not recommend escalating privileges unnecessarily in the documentation.  If it doesn't work without privilege, we should fix that bug instead of working around it.

File: README.md (hunk 1 above)
Comment (aftabsiddiqui) [reply README.md:15]: oh thats privilege issue was on my local system, I over looked that. No need for sudo for any of these operations.

**Review by ppaeps — APPROVED** [review]

**Comment by ppaeps** [comment]
I've merged these changes into main after rewriting history a bit.  Thanks for the patch!

**Comment by ppaeps** [comment]
This was rebased and merged as 7addd817c3492e87c3f0d654ff0470aa0627671b.  GitHub isn't smart enough to notice and auto-close this PR.

**Pull request closed without merging**